recipe/gen_patch_json.py

```diff
@@ -981,6 +981,10 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
         if i >= 0:
             deps[i] = "cudatoolkit >=11.2,<12.0a0"
 
+        if record_name == "cudatoolkit" and record.get('timestamp', 0) < 1681349920000:
+            cuda_major_minor = ".".join(record["version"].split(".")[:2])
+            record['depends'].append(f"cuda-version {cuda_major_minor}")
```

Comment:
We do not add new dependencies. It will result in loud warnings in existing environment. Please add this only to `constrains`.

Reply:
It's a good point. Fixed in a188b90.